requirements.viz.txt

```diff
@@ -0,0 +1,3 @@
+folium
+jupyter
+notebook
```

Comment:
Is this file being used in the GH Actions? Did a global search, did not see any mentioning of this file.